packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -275,11 +243,19 @@ describeNoCompat("GC Random tests", (getTestObjectProvider) => {
         }
         const container = getRandomContainer();
         const dataStore = await getRandomDataStoreFromContainer(container);
-        const parentDataStore = await getRandomParentForDataStore(dataStore, container);
-        if (parentDataStore === undefined) {
-            return;
+        const handle = await dataStore.generateRemoveHandleOp();
+        if (handle !== undefined) {
```

Comment:
should it always be defined ? Do we need to assert in case it is not ?

Reply:
We don't want to assert in the case it is not because there's no guarantee in my test that the DDS I'm trying to remove a handle from contains a handle.

Reply:
Made it so that the handle was always defined.